.travis.yml

```diff
@@ -6,8 +6,6 @@ branches:
 
 language: ruby
 rvm:
-  - 2.0.0
```

Comment:
Why the drop of support for Ruby 2.0 and 2.1?  Are you starting to use a 2.2+ only feature?